oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java

```diff
@@ -113,7 +114,13 @@ private void renew(EtcdClient client, String key, String json) {
             try {
                 client.refresh(key, KEY_TTL).send().get();
             } catch (Exception e) {
-
+                try {
+                    client.put(key, json).ttl(KEY_TTL)
+                        .setRetryPolicy(new RetryNTimes(1, 10))
```

Comment:
it looks good to me. Just one point that there is a loop already, I think it needs not another retryPolicy. What about you ?

Reply:
you are right,  I remove retryPolicy.